tests/test_app.py

```diff
@@ -299,11 +314,53 @@ def test_log_file_set(tmpdir, log_level, with_tty, regexes_required_lines, regex
     log_file = os.path.join(log_dir, log_file)
     with open(log_file, "r") as f:
         log_lines = f.readlines()
+        print(log_lines)
     check_logs(log_lines, regexes_required_lines, regexes_forbidden_lines, timed=True)
     check_logs(stdout, [], regexes_required_lines, timed=True)
     check_logs(stdout, [], regexes_required_lines, timed=False)
 
 
+@pytest.mark.parametrize_any(
+    "log_level, regexes_required_lines, regexes_forbidden_lines",
+    [
+        (
+            5,
+            [
+                r"[a-z.]*[ ]*INFO[\s]+[a-x\.A-Z]*[\s]Starting server endpoint",
+                r"[a-z.]*[ ]*DEBUG[\s]+[a-x\.A-Z]*[\s]Starting Server Rest Endpoint",
+            ],
+            [],
+        ),
+        (
+            4,
+            [
+                r"[a-z.]*[ ]*INFO[\s]+[a-x\.A-Z]*[\s]Starting server endpoint",
+                r"[a-z.]*[ ]*DEBUG[\s]+[a-x\.A-Z]*[\s]Starting Server Rest Endpoint",
+            ],
+            [],
+        ),
+        (
+            3,
+            [
+                r"[a-z.]*[ ]*INFO[\s]+[a-x\.A-Z]*[\s]Starting server endpoint",
+                r"[a-z.]*[ ]*DEBUG[\s]+[a-x\.A-Z]*[\s]Starting Server Rest Endpoint",
+            ],
+            [],
+        ),
+        (
+            2,
```

Comment:
Why did you exclude the case where `log_level=1`? That's an interesting case, because it verifies whether the `INFO` log line is filtered out.